<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Girdiler">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Agirlik">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Populasyon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MinAralik">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="MaxAralik">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Iterasyon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BireSayisi">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="GirdilerText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Girdiler</string>
    </property>
   </widget>
   <widget class="QLabel" name="AgirlikText">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ağırlık</string>
    </property>
   </widget>
   <widget class="QLabel" name="PopulasyonText">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Popülasyon Başına Çözüm Sayısı</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinAralikText">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxAralikText">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum Aralık</string>
    </property>
   </widget>
   <widget class="QLabel" name="IterasyonText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>İterasyon</string>
    </property>
   </widget>
   <widget class="QLabel" name="BireySayisiText">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Birey sayisi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="HesaplaButon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Hesapla</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>511</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
